app/assets/javascripts/components/sprint/SprintButton.js

```diff
@@ -0,0 +1,54 @@
+import React, {useState} from 'react'
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+
+export const SprintButton = ({calculatedSprintVelocity, sprintVelocity, onSimulate, onRevert}) => {
+  
+  const [simulatedSprintVelocity, setSimulatedSprint] = useState(sprintVelocity)
+  const [visible, setVisible] = useState(false)
+
+  return ReactDOM.createPortal(    
+    <>
+    <span id="velocity_value">
+      <i className="mi md-18" onClick={() => setVisible(!visible)}>trending_up</i>              
+      {sprintVelocity}                 
+    </span>
+    {visible && 
+        <div className="velocity-change">
```

Comment:
It seems to me these could be two components.

Reply:
Done, what do you think ?